fpr/templates/fpr/format/list.html

```diff
@@ -38,12 +38,12 @@
         <tbody>
         {% for format in formats %}
           <tr>
-            <td><a href="{% url 'format_detail' format.slug %}">{{ format.description }}</a></td>
+            <td><a href="{% url 'format_detail' format.slug %}">{{ format.description }} ({{ format.get_pronom_ids }})</a></td>
```

Comment:
This is working in most cases, but I've found entries like:

```
- WAV (, fmt/6)
- Dolby Digital Audio ()
- ...
```

This happens with enough frequency that I wonder if we should instead have a method in the model called `list_label` or similar (`<a>{{ format.list_label }}</a>`), returning a string with the formatting issues taken care of (empty strings, zero IDs, etc...). Or maybe this is a data sanitation task?